jupyter_server/extension/handler.py

```diff
@@ -12,19 +12,29 @@ def get_template(self, name):
         return self.settings[env].get_template(name)
 
 
-class ExtensionHandlerMixin():
-    """Base class for Jupyter server extension handlers. 
+class ExtensionHandlerMixin:
+    """Base class for Jupyter server extension handlers.
 
-    Subclasses can serve static files behind a namespaced 
-    endpoint: "/static/<extension_name>/" 
+    Subclasses can serve static files behind a namespaced
+    endpoint: "/static/<extension_name>/"
 
     This allows multiple extensions to serve static files under
-    their own namespace and avoid intercepting requests for 
-    other extensions. 
+    their own namespace and avoid intercepting requests for
+    other extensions.
     """
     def initialize(self, extension_name):
         self.extension_name = extension_name
 
+    @property
+    def extensionapp(self):
+        key = "{extension_name}".format(extension_name=self.extension_name)
```

Comment:
Does this do anything other than copy the string?

Reply:
Yuck! 😆 That was bad on my part. haha Fixed!